Mobile/CarWashService/LogoView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  LogoView.swift
+//  CarWashService
+//
+//  Created by Anna Boykova on 18/11/2019.
+//  Copyright © 2019 VXDESIGN.STORE. All rights reserved.
+//
+
+import SwiftUI
+
+struct LogoView: View {
```

Comment:
LogoAuthenticationView? I think we can add similar struct for app but with another view.